apps/server/src/modules/task/controller/dto/task-create.params.ts

```diff
@@ -11,4 +13,45 @@ export class TaskCreateParams {
 		nullable: false,
 	})
 	courseId!: string;
+
+	@IsString()
+	@IsMongoId()
+	@IsOptional()
+	@ApiPropertyOptional({
+		description: 'The id of an lesson object.',
+		pattern: '[a-f0-9]{24}',
+	})
+	lessonId?: string;
+
+	@IsString()
+	@SanitizeHtml()
+	@ApiProperty({
+		description: 'The title of the task',
+		required: true,
+	})
+	name!: string;
+
+	@IsString()
+	@IsOptional()
+	@SanitizeHtml(InputFormat.RICH_TEXT_CK5)
+	@ApiPropertyOptional({
+		description: 'The description of the task',
+	})
+	description?: string;
+
+	@IsDate()
+	@IsOptional()
+	@ApiPropertyOptional({
+		description: 'Date until the task is published',
```

Comment:
isnt availableDate the date since when the task is published, not "until"?

Reply:
will check